modules/alacritty/hm.nix

```diff
@@ -60,5 +50,26 @@ in
             };
           };
         };
-      };
+      }
+    )
+    (
+      { fonts }:
+      with fonts;
```

Comment:
Not sure this `with` is really needed anymore.

`monospace.name` isn't any better than `fonts.monospace.name` IMO, so it'd be nice to avoid having a `with`.